pkg/util/oom/oom_linux.go

```diff
@@ -75,6 +75,7 @@ func applyOOMScoreAdj(pid int, oomScoreAdj int) error {
 			err = fmt.Errorf("failed to apply oom-score-adj to pid %d (%v)", pid, err)
 			continue
 		}
+		defer f.Close()
```

Comment:
Let me see. Something to be discussed:
1. The for loop tries twice to write file. If we return error directly when the first time writing file fails, there would be no chance to try the next time.
2. According to 1, we should continue instead of return. If f.Write returns error both the twice trying, we only need to care the last error, and it will be returned by the last sentence `return err`. If f.Write returns error the first time and nil second, we should ignore the first error also.
3. It seems that we may not care about the result of f.Close() ?
/cc @smarterclayton . If there is something I misunderstand, feel free to point it out. Thanks!